setup/src/python/common/Dockerfile

```diff
@@ -12,7 +12,7 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-FROM ghcr.io/eclipse-velocitas/devcontainer-base-images/python:v0.3
+FROM ${{ devContainerBaseImageUriPrefix }}/python:${{ devContainerBaseImageTag }}
```

Comment:
we also have the language as variable, should we add this here? or directly in the manifest somehow?

In general I don't like that a user will need to know that cpp or python is still added to the URI. maybe someone just has one image for python but the name does not end with python...